src/inject/dynamic-theme/index.ts

```diff
@@ -329,12 +331,30 @@ function stopWatchingForUpdates() {
     stopWatchingForInlineStyles();
     removeDOMReadyListener(onDOMReady);
 }
+function checkMeta() {
```

Comment:
Every function should have a single purpose. So I would split it into 2 functions: `isAnotherDarkReaderInstanceActive` and `createDarkReaderInstanceMarker` (maybe there's a better word than `Marker`). And also it should remove the marker, when `removeDynamicTheme` is called.